README.md

```diff
@@ -35,21 +36,36 @@ or server in isolation will use the corresponding reference implementation to ve
 
 Below are the basic steps needed for setting up the suite to run against your implementation:
 
-1. The first step is to download the Conformance protos, which can be found on the Buf Schema Registry [here](https://buf.build/connectrpc/conformance).
-   From there, you will need to generate the code for the language you are testing.
+1. The first step is to access an SDK for the Conformance protos. These can be found on the Buf Schema Registry:
+   https://buf.build/connectrpc/conformance. You can download SDKs for some languages
+   [here](https://buf.build/connectrpc/conformance/sdks/main).
 
-2. Once complete, you will need to implement either the service, the client, or both (depending on which you are testing).
+   If you are using a language that is not supported by the BSR's selection of SDKs, you can generate one yourself
+   using the [`buf`](https://buf.build/docs/generate/tutorial) command-line tool.
+   After creating a [`buf.gen.yaml`](https://buf.build/docs/configuration/v1/buf-gen-yaml) file, to configure the
+   code generation, you'll then run `buf generate buf.build/connectrpc/conformance`.
+
+2. Once you have an SDK, with generated code for the
+   [Conformance Service](https://buf.build/connectrpc/conformance/docs/main:connectrpc.conformance.v1#connectrpc.conformance.v1.ConformanceService)
+   and related messages, you will need to implement either the service, the client, or both (depending on which you are testing).
    To do so, follow the instructions specified in the
```

Comment:
We'll need to replace these sentences later, with links to dedicated guides for creating a client-under-test and server-under-test, once those pages exist.